llvm/lib/Passes/PassBuilder.cpp

```diff
@@ -947,6 +948,32 @@ parseLowerAllowCheckPassOptions(StringRef Params) {
   return Result;
 }
 
+Expected<MetaRenamerOptions> parseMetaRenamerPassOptions(StringRef Params) {
+  MetaRenamerOptions Result;
+  while (!Params.empty()) {
+    StringRef ParamName;
+    std::tie(ParamName, Params) = Params.split(';');
+    bool Enable = !ParamName.consume_front("no-");
+    if (ParamName == "rename-only-inst") {
+      Result.RenameOnlyInst = Enable;
+    } else if (ParamName.consume_front("rename-exclude-struct-prefixes=")) {
+      Result.RenameExcludeStructPrefixes = ParamName;
+    } else if (ParamName.consume_front("rename-exclude-global-prefixes=")) {
+      Result.RenameExcludeGlobalPrefixes = ParamName;
+    } else if (ParamName.consume_front("rename-exclude-alias-prefixes=")) {
+      Result.RenameExcludeAliasPrefixes = ParamName;
+    } else if (ParamName.consume_front("rename-exclude-function-prefixes=")) {
+      Result.RenameExcludeFunctionPrefixes = ParamName;
+    } else {
+      return make_error<StringError>(
+        formatv("invalid MetaRenamer pass parameter '{0}' ", ParamName)
```

Comment:
```suggestion
        formatv("invalid metarenamer pass parameter '{0}' ", ParamName)
```
These should probably refer to the form used in the passes format

Reply:
fixed